plugin.video.regiotv/resources/lib/addon.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+''' This is the actual Regio TV video plugin entry point '''
+
+from __future__ import absolute_import, division, unicode_literals
+from routing import Plugin
+from data import CHANNELS
+
+plugin = Plugin()  # pylint: disable=invalid-name
+
+
+@plugin.route('/')
+def main_menu():
+    ''' The Regio TV plugin main menu '''
+    from xbmcaddon import Addon
+    from xbmcgui import ListItem
+    import xbmcplugin
+
+    xbmcplugin.setContent(plugin.handle, content='files')
+    xbmcplugin.setPluginCategory(plugin.handle, category='Zenders')
+    xbmcplugin.setPluginFanart(plugin.handle, image=Addon().getAddonInfo('fanart'))
+    xbmcplugin.addSortMethod(plugin.handle, sortMethod=xbmcplugin.SORT_METHOD_UNSORTED)
+    xbmcplugin.addSortMethod(plugin.handle, sortMethod=xbmcplugin.SORT_METHOD_LABEL)
+
+    listing = []
+    for channel in CHANNELS:
+        is_folder = False
+        is_playable = True
+
+        label = '{name} [COLOR gray]| {label}[/COLOR]'.format(**channel)
+        plot = '[B]{name}[/B]\nRegio {label}\n\n[I]{description}[/I]\n\n[COLOR yellow]{website}[/COLOR]'.format(**channel)
+        list_item = ListItem(label=label, label2=channel.get('label'), offscreen=True)
+        list_item.setProperty(key='IsInternetStream', value='true' if is_playable else 'false')
```

Comment:
I don't think that we know such a property for ListItems? @enen92

Reply:
I am not sure we should allow this to be enforced from the add-on. This is an internal property that Kodi sets depending on the stream. Better to set the mimetype or find why Kodi doesn't set it for the stream and either fix Kodi or explain why this is needed

Reply:
Removed.